message_tagging_service/consumer.py

```diff
@@ -132,7 +132,14 @@ def _consumer_wrapper(msg, data=None):
         private_key=conf.rhmsg_private_key,
         trusted_certificates=conf.rhmsg_ca_cert,
     )
-    consumer.consume(conf.rhmsg_queue, callback=_consumer_wrapper)
```

Comment:
Since the default for `rhmsg_subscription_name` is `None`, why not do the following?
```python
consumer.consume(conf.rhmsg_queue, callback=_consumer_wrapper, subscription_name=conf.rhmsg_subscription_name)
```

Reply:
Yes, of course! Done :)